requirements-dev.txt

```diff
@@ -5,6 +5,7 @@ codecov
 coverage
 flake8
 pytest >=3.9
+pyPDB
```

Comment:
Yeah, this has broken a lot of suitcases and I've been lazy about sorting out a fix. I think we can fix it upstream (in caproto, or ophyd) rather than having to touch all the suitcases---and touch them again later when caproto changes again, etc.